keps/NNNN-kep-template/README.md

```diff
@@ -452,6 +453,36 @@ You can take a look at one potential example of such test in:
 https://github.com/kubernetes/kubernetes/pull/97058/files#diff-7826f7adbc1996a05ab52e3f5f02429e94b68ce6bce0dc534d1be636154fded3R246-R282
 -->
 
+### Testing Quality
+
+<!--
+This section must be completed when targeting alpha to a release.
+-->
+
+###### What is the current (unit) test coverage of packages that will be touched when implementing this KEP?
```

Comment:
this seems hard to answer ahead of implementation, and likely to become stale quickly

Reply:
Instead of these few questions, how about:

[ ] I understand the owners of the involved components may require updating existing tests to make this code solid enough to build on top of (everyone has to check yes)

and 

[ ] statement from existing owners regarding pre-existing component test health (to be supplied by KEP reviewers)

Reply:
> this seems hard to answer ahead of implementation, and likely to become stale quickly

Thinking about it, I will tweak to the following:
- this part will be required only for Beta/GA and will have to report healthiness of packages that were touched - this means that you won't need to predict anything (but rather report) and will be a double check of both authors and reviewers did a reasonable job.

> Instead of these few questions, how about:

I'm not against adding those, but I don't believe in those being enough. We need some mechanism to track it this too.

Reply:
The only thing I feel strongly about is to give the area owners the chance / responsibility to state the testing deficiencies of the existing code. I think having some human judgment in there is going to be better than picking a coverage target, for example. We can suggest a coverage target if the owners don't have a better idea about the deficiencies.

Reply:
Which I mentioned in the template explicitly too.

As an approver you will be able to say that 20% coverage is fine for you if you want.

But the first thing to achieve is to improve visibility and awareness - many people don't realize how bad our coverage is in some areas.